src/contracts/Input/Parser/Query/Criterion/BaseCriterionProcessor.php

```diff
@@ -4,6 +4,7 @@
  * @copyright Copyright (C) Ibexa AS. All rights reserved.
  * @license For full copyright and license information view LICENSE file distributed with this source code.
  */
+
```

Comment:
Since we are touching it anyway:
```suggestion
declare(strict_types=1);
```

Reply:
Actually we are not, not a part of this PR